setup.py

```diff
@@ -42,6 +42,7 @@ def read(filename):
         'django-mptt>=0.6.0',
```

Comment:
django-mptt 0.7.1 has been released a few days ago. If I understand it well, this was blocking for this PR.